localtools/src/integrationTest/kotlin/testconfiguration/util/AppResources.kt

```diff
@@ -0,0 +1,69 @@
+package testconfiguration.util
+
+import io.provenance.classification.asset.util.enums.ProvenanceNetworkType
+import io.provenance.classification.asset.util.wallet.ProvenanceAccountDetail
+
+object AppResources {
+    // IMPORTANT: This version should be pinned to the latest asset classification smart contract instance that the client
+    // and libs are compatible with.  Local testing for unpublished versions of the smart contract's WASM file can be
+    // done by tweaking ManagedProvenanceTestContainer's afterStartup function implementation to fetch the file from local
+    // resources, if needed.
+    const val CONTRACT_VERSION = "1.0.8"
```

Comment:
Not sure if I should do this value pin or not.  It feels right to forcibly update the value, and I have comments about it elsewhere, but this could get lost amid the shuffle maybe